package.json

```diff
@@ -119,7 +119,7 @@
     "@typescript-eslint/eslint-plugin": "^4.14.1",
     "@typescript-eslint/parser": "^4.23.0",
     "@wojtekmaj/enzyme-adapter-react-17": "^0.6.0",
-    "add-asset-html-webpack-plugin": "^3.2.0",
+    "add-asset-html-webpack-plugin": "^2.1.2",
```

Comment:
Why the downgrade?

Reply:
That's strange indeed

Reply:
that was the result from npm audit fix.. I have updated the add-asset-html-webpack-plugin. However, putting back the eslint-nibble update was not successful.